src/Orleans.CodeGeneration.Build/Orleans.CodeGeneration.Build.csproj

```diff
@@ -20,6 +20,7 @@
     <PublishDir>$(PublishRoot)$(TargetFramework)</PublishDir>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
+    <DevelopmentDependency>true</DevelopmentDependency>
```

Comment:
Unfortunately NuGet tooling does not yet honor this on restore, but eventually they will, so it makes sense to add it anyway. Tracking issue: https://github.com/NuGet/Home/issues/4125